Algorithm/QCAlgorithm.Python.cs

```diff
@@ -50,11 +50,12 @@ public void SetPandasConverter()
         /// </summary>
         /// <param name="type">Data source type</param>
         /// <param name="symbol">Key/Symbol for data</param>
+        /// <param name="accountCurrency">Account currency</param>
         /// <param name="resolution">Resolution of the data</param>
         /// <returns>The new <see cref="Security"/></returns>
-        public Security AddData(PyObject type, string symbol, Resolution resolution = Resolution.Minute)
+        public Security AddData(PyObject type, string symbol, string accountCurrency, Resolution resolution = Resolution.Minute)
```

Comment:
I don't think we should be updating the `AddData` methods to accept account currency. The account currency can be resolved via `Portoflio.CashBook.AccountCurrency`